Makefile

```diff
@@ -37,15 +44,20 @@ embedmd:
 docs: embedmd
 	embedmd -w `find Documentation -name "*.md"`
 
-assets: gobindata
+bindata: pkg/manifests/bindata.go
+pkg/manifests/bindata.go: $(ASSETS) $(GOBINDATA)
 	# Using "-modtime 1" to make generate target deterministic. It sets all file time stamps to unix timestamp 1
-	go-bindata -mode 420 -modtime 1 -pkg manifests -o pkg/manifests/bindata.go assets/...
+	go-bindata -mode 420 -modtime 1 -pkg manifests -o $@ assets/...
+
+$(ASSETS): $(JSONNET)
+	./hack/build-jsonnet.sh
 
 generate:
 	docker build -t tpo-generate -f Dockerfile.generate .
-	docker run --rm  --security-opt label=disable -v `pwd`:/go/src/github.com/openshift/cluster-monitoring-operator -w /go/src/github.com/openshift/cluster-monitoring-operator tpo-generate make merge-cluster-roles assets docs
+	docker run --rm --security-opt label=disable -v `pwd`:/go/src/github.com/openshift/cluster-monitoring-operator -w /go/src/github.com/openshift/cluster-monitoring-operator tpo-generate make merge-cluster-roles bindata docs
 
-gobindata:
+gobindata: $(GOBINDATA)
```

Comment:
this way we can know when we need go-bindata installed and avoid re-generating static assets if go-bindata was already present